### DISCUSSION TITLE
Convert SVG to PNG with image attribute inside svg file

### DISCUSSION THREAD

**Discussion by yuliazarkh** [dlemstra/Magick.NET#584] [General] [answered]
### System Configuration
- Magick.NET version: Magick.NET-Q16-AnyCPU - Version=7.15.3
- Environment : Windows 10, .NET Core 3.1

### Question
When trying to render SVG file that includes image attribute with link to image in the web, to PNG, it renders to without image inside . Does the library support it?
Input:
[testImageSvg-input.txt](https://github.com/dlemstra/Magick.NET/files/4271643/testImageSvg-input.txt)

Same issue happens with image data url inside the svg file.
For Example:
`<image width="1200px" height="1450px" preserveAspectRatio="xMidYMid slice" xmlns:ns1="http://www.w3.org/1999/xlink" ns1:href="data:image/png ..."/>`

Both cases render to png without image inside:
![imageTestResult](https://user-images.githubusercontent.com/47394331/75623939-29b41800-5bb8-11ea-8f82-dd23c0971476.png)

Thanks for help!

**Comment by dlemstra** [comment, marked as answer]
Downloading of external images from inside a svg file is not allowed in this library.
